File: basic_python/HELLOJAVA/src/day07/mySelect.java
package day07;

import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.PreparedStatement;
import java.sql.SQLException;
import java.sql.ResultSet;  
import java.sql.Statement;


public class mySelect {
		
	public static void main(String[] args) throws Exception {
		

		String driver = "oracle.jdbc.driver.OracleDriver";
		String url = "jdbc:oracle:thin:@localhost:1521:xe"; //oracle expression(평가판)버전은 xe / 회사에서는 orcl이 대부분
		String sql = "select col01, col02, col03 from sample";
		
		Class.forName(driver);
		Connection conn = DriverManager.getConnection(url, "python", "python");
		Statement stmt = conn.createStatement();
		ResultSet rs = stmt.executeQuery(sql);
		
		//rs.next() : 뒤에 데이터가 있는지 여부를 물음
		
		
		while (rs.next()) {
			String col01 = rs.getString("col01");
			String col02 = rs.getString("col02");
			String col03 = rs.getString("col03");
			
			System.out.print("col01 : "+ col01);
			System.out.print(" col02 : "+ col02);
			System.out.println(" col03 : "+ col03);
		}
		
		stmt.close();
		conn.close();
		

	}
}

File: basic_python/HELLOPYTHON/day08/myselect.py
import cx_Oracle

#연결에 필요한 기본 정보 (유저, 비밀번호, 데이터베이스 서버 주소)
#('유저/비밀번호@데이터베이스서버주소')
connection = cx_Oracle.connect('python','python','localhost:1521/xe')
cursor = connection.cursor()

sql_str = 'select col01, col02, col03 from sample'
cursor.execute(sql_str)

for name in cursor:
    print("테스트 이름 리스트 : ", name[0])

cursor.close()
connection.close()